test/TestHelper.cpp

```diff
@@ -172,6 +201,74 @@ void ImportTest::importTransaction(json_spirit::mObject& _o)
 	}
 }
 
+void ImportTest::checkExpectedState(State const& _stateExpect, State const& _statePost, WhenError _throw)
```

Comment:
you do not need this function when you use default values for ImportTest::checkExpectedState in line 210